index.html

```diff
@@ -6,11 +6,211 @@
+                           </div>
+                        </section> -->
+
+                    </section>
+                </section>  
+                
+                <section class="menu-header">
+                    <nav class="navigation-bar">
+                        <ul>
+                            <li class="navigation-item">
+                                    <a class="navigation-link" href="#">HOME</a>
+                            </li>
+                            <li class="navigation-item">
+                                <a class="navigation-link" href="#">CAKES</a>
+                            </li>
+                            <li class="navigation-item">
+                                <a class="navigation-link" href="#">ORDERING</a>
+                            </li>
+                            <li class="navigation-item">
+                                <a class="navigation-link" href="#">LESSONS</a>
+                            </li>
+                            <li class="navigation-item">
+                                <a class="navigation-link" href="#">ABOUT US</a>
+                          </li>
+                        </ul>
+                    </nav>
+                </section>                    
+        </section>
+
+        <section class="content-box">
+                <p class="mobile-prompt">
+                    We make all types of same-day cakes from Cupcakes,
+                    </br>Celebration Cakes, Birthday Cakes, Chocolate cakes,
+                    </br>Novelty cakes, Bespoke cakes and even same-day Wedding Cakes.
+                    <br/><br/>If you can dream of it we can make it.<br/><br/>
+
+                     <!-- &#8595; is the symbol for a down-pointing arrow-->
+                    &#8595;Hover over the images below for more information&#8595;
+                </p>
+                
+                <p class="tablet-prompt">
+                    We make all types of same-day cakes from
+                    </br> Cupcakes, Celebration Cakes, Birthday Cakes,
+                    </br> Chocolate cakes, Novelty cakes, Bespoke cakes and even same-day Wedding Cakes.
+                    <br/><br/>If you can dream of it we can make it.<br/><br/>
+                    
+                    Hover over the image to the left and the images below for more information
+                </p>
+
+                <p class="desktop-prompt">
+                    We make all types of same-day cakes
+                    </br> from Cupcakes, Celebration Cakes,
+                    </br> Birthday Cakes, Chocolate cakes, Novelty cakes, Bespoke cakes
+                    </br> and even same-day Wedding Cakes.
+                    <br/><br/>If you can dream of it we can make it.<br/><br/>
+                    
+                    Hover over the image to the left and the images below for more information
+                </p>
+
```

Comment:
Why are you repeating all this content 3 times here? You only need it once. If you need to adapt the styling for each device, use media queries to change the style.

Secondly, do not use `<br>` to force the content onto the next line. Remove all the `br`s and let the content flow naturally and adapt to the space it has on each screen size.